lib/models/Player.js

```diff
@@ -328,6 +353,20 @@ Player.prototype.unMute = function unMute() {
     { mute: 0 });
 };
 
+Player.prototype.unMuteGroup = function unMuteGroup() {
+  return soap.invoke(
+      `${this.baseUrl}/MediaRenderer/GroupRenderingControl/Control`,
+      TYPE.GroupMute,
+      { mute: 0 });
+};
+
+Player.prototype.muteGroup = function muteGroup() {
+  return soap.invoke(
+      `${this.baseUrl}/MediaRenderer/GroupRenderingControl/Control`,
+      TYPE.GroupMute,
+      { mute: 1 });
+};
+
```

Comment:
Aha, never mind, seems like mute is actually non-conforming compared to repeat/shuffle/crossfade. My bad.